charts/steward/test/cm_test.go

```diff
@@ -0,0 +1,78 @@
+// +build helm
+
+package test
+
+import (
+	"log"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"gotest.tools/assert"
+	v1 "k8s.io/api/core/v1"
+)
+
+func Test_ConfigPipelineruns(t *testing.T) {
+	t.Parallel()
+	template := "templates/config-pipelineruns.yaml"
+
+	for _, tc := range []struct {
+		name               string
+		values             map[string]string
+		expectedMapEntries map[string]string
+		expectedError      string
+	}{
+		{"full",
+			map[string]string{
+				"pipelineRuns.jenkinsfileRunner.image":           "repo1:tag1",
+				"pipelineRuns.jenkinsfileRunner.imagePullPolicy": "policy1",
+			},
+			map[string]string{
+				"jenkinsfileRunner.image":           "repo1:tag1",
+				"jenkinsfileRunner.imagePullPolicy": "policy1",
+			},
+			"",
+		},
+		{"imageOnly",
+			map[string]string{
+				"pipelineRuns.jenkinsfileRunner.image": "repo1:tag1",
+			},
+			map[string]string{
+				"jenkinsfileRunner.image":           "repo1:tag1",
+				"jenkinsfileRunner.imagePullPolicy": "IfNotPresent",
+			},
+			"",
+		},
+
+		{"old",
+			map[string]string{
+				"pipelineRuns.jenkinsfileRunner.image.repository": "repo1",
+				"pipelineRuns.jenkinsfileRunner.image.tag":        "tag1",
+				"pipelineRuns.jenkinsfileRunner.image.pullPolicy": "policy1",
+			},
+			map[string]string{},
+			"exit status 1",
```

Comment:
Does old (existing) configuration work or not? If this change breaks compatibility we need to increase at least the minor version and mention the breaking change in the changelog.

Reply:
Old existing configuration will not work anymore. This is mentioned in the updateNote.
Change impact is set to 'minor'.
Version can only be adjusted shortly before the merge to master.

Reply:
If old configuration does not work anymore the impact is 'major'. Since we have a 0.x version the minor version needs to be increased, compared to the latest released version.

> Version can only be adjusted shortly before the merge to master.

Wrong. The version needs to be adjusted in the PR (if necessary) in the Chart.yaml.

Reply:
Version is already 0.6.0-dev (latest release 0.5.x), so no need to increase it further, already done.